src/module/EMDoublePairProduction.cpp

```diff
@@ -8,9 +8,10 @@
 
 namespace crpropa {
 
-EMDoublePairProduction::EMDoublePairProduction(PhotonField photonField, bool haveElectrons, double limit) {
+EMDoublePairProduction::EMDoublePairProduction(PhotonField photonField, bool haveElectrons, double thinning, double limit) {
```

Comment:
Do old scripts, using only photonField, haveElectrons, and limit as arguments still run?

Reply:
The old scripts run as usual for the photonField and haveElectrons arguments. But I had to make a decision concerning the limit argument. Since few people set the limit anyway, I added the thinning argument BEFORE the limit to make it more accessible. This would break scripts that explicitly use this argument.

Reply:
I agree that most people do not set these arguments but would it be possible to add a meaningfull warning, when these arguments are set? 

If not the user will probably figure out fast that a new arguments was introduced and is able to adopt his code based on the default error message.

Reply:
@lukasmerten : do you think this is a problem?
Do you use the limit argument?

Reply:
No I don't think this is a problem and I am also not setting this values explicitly. So it is probably fine as it is.